underactuated.html

```diff
@@ -788,7 +803,7 @@
 
 <li> Legged robots are underactuated.  Consider a legged machine with $N$
 internal joints and $N$ actuators.  If the robot is not bolted to the ground,
-then the degrees of freedom of the system include both the internal joints and
+then the degrees of freedom (DOFs) of the system include both the internal joints and
```

Comment:
true.  but we never defined "robot" or "joints" either.  I think it's clear enough.